### PR TITLE
az-advisor: add page

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]

**Comment by gutjuri** [comment]
Thanks for your contribution!

File: pages/common/az-advisor.md
Comment (kbdharun) [inline pages/common/az-advisor.md:4]:
```suggestion
> Part of `az`.
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

Edit. If possible, can you enable the allow maintainers to edit the pull request option? (It allows us to make trivial changes like the one below before merging the PR).

**Pull request merged**
